dtfs-central-api/api-tests/v1/tfm/facilities/tfm-facilities-get-facilities-for-deal.api-test.js

```diff
@@ -25,9 +25,8 @@ describe('/v1/tfm/deals/:id/facilities', () => {
   describe('GET /v1/tfm/deal/:id/facilities', () => {
     it('returns a 400 error if the id is not a valid mongo id', async () => {
       const invalidMongoId = 'abc';
-      const { status, body } = await testApi.get(`/v1/tfm/deals/${invalidMongoId}/facilities`);
+      const { status } = await testApi.get(`/v1/tfm/deals/${invalidMongoId}/facilities`);
```

Comment:
any reason for this>

Reply:
Because the test only specifies that we return a 400, and the actual response body comes from express-validator should not really matter for the sake of the test (I also really dislike the `res.status(400).send({ status: 400, message: '...' })` pattern)

Reply:
Do you think we should test the error message anyway, in case it's used in future downstream?

Reply:
Not at the api-test level - I think the status communicates more to us than the body of the error at the api-test level, and testing the body of the error would be a test of the express-validator package rather than a test of our implementation

Reply:
I'd somewhat agree except `400` is generic, and there has been issues on this codebase where tests similar to this are passing but really its an issue with the test configuration.

I get what you mean that it should be tested in the express validator package, I think that is a fair point... I'd still like to see it to document it's working as intended though?

Reply:
Discussed offline - work to assure the correct errors are thrown will be addressed in a follow-up PR